### PR TITLE
Update for origami v2 on npm 

### PR DESCRIPTION
**Pull request by chee**
## notes

<ul><li>First, run the app locally in its current state.<ul><li>Use the version of npm the app expects right now</li></ul></li><li>Get the app up-to-date<ul><li>Upgrade to the latest major versions of any Origami bower components you are using</li><li>Get the app running</li></ul></li><li>If the app uses npm@6:<ul><li>clean the directory &nbsp;(delete any built artifacts)</li><li>switch to npm@7</li><li>run the app again</li><li>sometimes this doesn't work because npm 7 contains multitudes so you need to delete <code>package-lock.json</code> and try again</li></ul></li><li>Clean the directory (remove any built artifacts)</li><li>Install all your origami dependencies as npm dependencies</li></ul><p>fish(1):</p><pre><code class="language-text-x-sh">for dep in (jq -r '.dependencies|keys[]' bower.json);
      npm install @financial-times/$dep@prerelease
end</code></pre><p>bash(1) or zsh(1):</p><pre><code class="language-text-plain">for dep in $(jq -r '.dependencies|keys[]' bower.json); do
      npm install @financial-times/$dep@prerelease
done</code></pre><ul><li><code>rm .bowerrc bower.json</code></li><li><code>rm -r bower_components</code></li><li><code>npm uninstall bower</code></li><li>Update your sass <code>includePath</code> to include <code>node_modules</code></li><li>Either:<ul><li>Update your sass <code>includePath</code> to also include <code>node_modules/@financial-times</code></li><li>Or, update your sass imports from, for example, <code>o-icons</code> to&nbsp;<code>@financial-times/o-icons</code></li></ul></li><li>Update your origami javascript imports from, for example <code>"o-tracking"</code> to <code>"@financial-times/o-tracking"</code><ul><li><code>s/import ‘o/import ’@financial-times/o-/</code></li><li><code>s/from ‘o/from ’@financial-times/o-/</code></li></ul></li></ul>

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,24 +16,45 @@
...
-    "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=bower_components --load-path=. | postcss --use autoprefixer --output public/main.css --map",
-    "build:js": "mkdir -p public && oax ./src/main.js --for-bower | swc -f - -C minify > public/main.js",
+    "build:css": "mkdir -p public && sass src/main.scss  --style=compressed --load-path=node_modules --load-path=node_modules/@financial-times --load-path=. | postcss --use autoprefixer --output public/main.css --map",
+    "build:js": "mkdir -p public && oax ./src/main.js | swc -f - -C minify > public/main.js",
     "build": "npm-run-all --parallel build:js build:css",
     "watch:css": "nodemon --watch src -e scss -x 'npm run build:css'",
     "watch:js": "nodemon --watch src -e js -x 'npm run build:js'",
     "watch": "npm-run-all --parallel watch:js watch:css",
-    "heroku-postbuild": "bower install && npm run build",
+    "heroku-postbuild": "npm install && npm run build",
```
Comment (JakeChampion) [inline package.json:31]: We don't need to have the `npm install` part here, Heroku does a full install (dev-dependencies) before running the `heroku-postbuild` script and prunes the dependencies after the script finishes -- https://devcenter.heroku.com/articles/nodejs-support#package-installation

```suggestion
    "heroku-postbuild": "npm run build",
```

**Review by JakeChampion — APPROVED** [review]

**Comment by JakeChampion** [comment]
When running this locally, the homepage looks to no longer have checkboxes for the search filter for origami type -- 
image

**Comment by chee** [comment]
> When running this locally, the homepage looks to no longer have checkboxes for the search filter for origami type --
> image

nice

**Comment by chee** [comment]
> When running this locally, the homepage looks to no longer have checkboxes for the search filter for origami type --
> image

sorry @JakeChampion, can't reproduce. mine has radio buttons locally:
image

**Comment by chee** [comment]
ah, this seems to have fixed it:
image

**Review by JakeChampion — APPROVED** [review]

**Comment by origamiserviceuser** [comment]
:tada: This PR is included in version v219.1.0 :tada:

**Pull request merged**